src/OSPSuite.R/MinimalImplementations/FullPathDisplayResolver.cs

```diff
@@ -0,0 +1,27 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.R.MinimalImplementations
+{
+   public class FullPathDisplayResolver : IFullPathDisplayResolver
+   {
+      private readonly IEntityPathResolver _entityPathResolver;
+
+      public FullPathDisplayResolver(IEntityPathResolver entityPathResolver)
+      {
+         _entityPathResolver = entityPathResolver;
+      }
+
+      public string FullPathFor(IObjectBase objectBase, bool addSimulationName = false)
```

Comment:
Looks like a real implementation to me that could be moved to core. But good for now

Reply:
Don't we have something like this already in core?

Reply:
In fact no. We haev the same implementation in MoBi but PKSim uses something different using the representationInfo repository. If we move that concept out of PKSim and into core, and have MobI simple return name for display name, then we could reuse the same logic. Might do it next